templates/views/index.html

```diff
@@ -1,5 +1,5 @@
 {% extends 'layouts/layout.html' %}
 
 {% block content %}
-{% include 'content/overview.html' %}
-{% endblock %}
\ No newline at end of file
+I'm a home page :D
```

Comment:
Note: Just wanted to take the overview page out of here because that was linking to the overview page with hardwired rogue strings. I'm sure you will have a specific home page, and I didn't want to think I was on the wrong page =P